src/core/logging.js

```diff
@@ -1,23 +1,28 @@
+var listeners = {};
 var loggingCallbacks = {};
 
-// Register logging callbacks
-function registerLoggingCallbacks( obj ) {
-	var i, l, key,
-		callbackNames = [ "begin", "done", "log", "testStart", "testDone",
+// DEPRECATED: Register old logging callbacks
+// This will be removed on QUnit 2.0.0+
+// From now on use the QUnit.on( eventType, callback ) format
+function registerLoggingCallbacks( QUnit ) {
+	var i, l, key;
+	var callbackNames = [ "begin", "done", "log", "testStart", "testDone",
 			"moduleStart", "moduleDone" ];
+	var dictionary = {
```

Comment:
I think this should reference https://github.com/js-reporters/js-reporters#event-names